samples/features/sql-big-data-cluster/deployment/private-bdc/scripts/deploy-private-aks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Define a set of environment variables to be used in resource creations.
+
+export REGION_NAME=northeurope
```

Comment:
This is missing the subscription id input? Also make the region and the resource group name as input parameters.

Reply:
This one doesn't need sub id since no UDR involved ( I am leaving the choice to the customers since last time we spoke ), I think this is heads down the best way I could approach.